src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  lastPressedKey: string
+}
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    lastPressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.currentPressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.currentPressedKey);
+  }
+
+  currentPressedKey = (event: KeyboardEvent) => {
```

Comment:
Method's name better start with a verb (so you could easily tell what your method actually do)
 ;)